TestColorMap.cpp

```diff
@@ -0,0 +1,39 @@
+// test_colormap.cpp
+#include <gtest/gtest.h>
+#include <sstream>
+#include <string>
+#include "ColorMap.h"
+
+TEST(ColorMapTest, ColorMapOutput) {
+    std::vector<std::string> result = getColorMap();
+    std::vector<std::string> expected_output = {
+        "Index=0, MajorColor=White, MinorColor=Blue",

```

Comment:
Is `Index` meant to be the pair number? That's supposed to go from 1 to 25, not 0 to 24, right?
Also, is the output readable by the user? There's a lot of repetition of PascalCase words, which may not be familiar to a person reading the report.
Let the test capture (and safeguard) what's important for the consumer of the code